examples/tut-6-rpc/docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
     depends_on:
       - algorand-postgres-db
     environment:
-      - REACH_DEBUG
+      - REACH_DEBUG=1
```

Comment:
We actually want this to be what it was before, because it means that `REACH_DEBUG` from the user will be inherited

Reply:
Ah, okay - thanks.

Reply:
Fixed in d9a6aa6